client/src/components/dataList/IndividualsInfo.js

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+import React from "react";
+
+const IndividualsInfo = () =>{
+    const [individuals, setIndividuals] = useState([]);
+    
+    //get individuals data table
+    const getIndividuals = async () =>{
+        const response = await fetch(
+            `http://localhost:4040/individuals`
+        );
+        const individual = await response.json();
+        console.log(individual );
+        setIndividuals(individual );
+    };
+
+    useEffect(() => {
+        getIndividuals();
+    }, []);
+
+    return(
+<>
+<header> Individuals Data Table </header>
```

Comment:
Fix your indentation here. This is quite hard to read and that makes it hard to understand.